website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -0,0 +1,381 @@
+---
```

Comment:
AFAICT the review for this file is just the new name, all the content is still the old layout.

Reply:
This is the idea. Just add the quick start and rename the old one to tutorial.